### DISCUSSION TITLE
Updating and Uninstalling

### DISCUSSION THREAD

**Discussion by raj23689** [metafates/mangal#19] [Q&A] [answered]
Is there any way to update and uninstall the program ?

Like I installed `mangal` using the `go` method , so now how will I update and uninstall it in the future ?

**Comment by metafates** [comment, marked as answer]
If you've used go method you can update mangal by running `go install github.com/metafates/mangal@latest`

To uninstall you have to just delete binary file. If you're not sure where it's located run `which mangal` (`where.exe mangal` for powershell)

For example:
- Bash/Zsh - `rm $(which mangal)`
- Fish - `rm (which mangal)`
- Powershell - `rm $(where.exe mangal)`

**Reply by raj23689** [reply]
Thanks, should I update the ```readme.md``` with the above info ?

**Reply by metafates** [reply]
Don't worry, I'll do it myself when I get home. Or you can make a PR or something if you want to 🥸
